src/plugins/intel_cpu/tools/commit_slider/utils/helpers.py

```diff
@@ -635,6 +682,7 @@ def __init__(self, dict: dict = None):
             for k, v in dict.items():
                 setattr(self, k, v)
 
+
```

Comment:
please remove redudant lines

Reply:
All blank lines assigned according to our python syntax requirements. Out of class functions divided by **two** blanklines